homepage.css

```diff
@@ -144,54 +155,212 @@ ul li:hover ul{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images1 img{
+	height:214px;
+	margin-left:25px;
 }
 
 .images2{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images2 img{
+	height:214px;
+	margin-left:25px;	
 }
 
 .images3{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
```

Comment:
31/08/2021: Setting the div box that hold the image that act as a link to a small query page.